### DISCUSSION TITLE
Disable keylist feature?

### DISCUSSION THREAD

**Discussion by BearAndBunny** [fabiocaccamo/python-benedict#412] [Q&A]
Is it possible to disable the keylist feature?

Using tuples as keys with the usual syntax does not work:
```
from benedict import benedict

x = benedict()
x[("a", True)] = "test"
print(x)

x = benedict()
x.update({("a", True): "test"}) #works, but clumsy syntax
print(x)

y = dict()
y[("a", True)] = "test"
print(y)
```
Output:
```
{'a': [None, 'test']}
{('a', True): 'test'}
{('a', True): 'test'}
```

**Comment by fabiocaccamo** [comment]
@BearAndBunny thank you for reporting this, actually the keylist is a core feature, however the behaviour should be consistent across all methods.

**Reply by BearAndBunny** [reply]
The keylist feature could be changed to work only with lists and not with tuples. Lists are not hashable and cannot be used as keys with a dict(). That way it would be consistent and benedict uses lists as keys in a special way as a feature.

**Reply by fabiocaccamo** [reply]
Yes, this could be a nice improvement indeed!

**Comment by fabiocaccamo** [comment]
@BearAndBunny fixed in `0.34.0` version.
